google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQuery.java

```diff
@@ -850,18 +850,6 @@ public int hashCode() {
    */
   boolean delete(DatasetId datasetId, DatasetDeleteOption... options);
 
-  /**
-   * Deletes the requested table.
-   *
-   * @deprecated Now that BigQuery datasets contain multiple resource types, this invocation is
-   *     ambiguous. Please use more strongly typed version of {@code #delete} that leverages an
-   *     non-ambiguous resource type identifier such as {@code TableId}.
-   * @return {@code true} if table was deleted, {@code false} if it was not found
-   * @throws BigQueryException upon failure
-   */
-  @Deprecated
-  boolean delete(String datasetId, String tableId);
```

Comment:
This method is not an internal method. It is exposed to the client, Are you sure you can remove this method?

Reply:
this method no longer used in project so removed that due to deprecated it's showing as warning.

Reply:
But it's the surface interface, and we can't assume no end-user would be using it.

Reply:
I think you can we can remove this deprecated method in major version bump only